.github/workflows/ci_integrations_actions.yml

```diff
@@ -91,10 +91,40 @@ jobs:
           name: ${{ matrix.output_name }}
           path: metrics/ci_integrations/${{ matrix.output_name }}
           if-no-files-found: error
+  
+  ci-integrations-linux-generate-sha256-actions:
```

Comment:
I believe it would be better to generate the checksum in the job where we've built the application to avoid the situation when we download the tool with errors and generate the checksum for such a file.